.github/workflows/build_workflow.yml

```
@@ -152,7 +152,7 @@ jobs:
           git commit -m "Update documentation" -a || true
 
       - name: Push Changes
-        uses: ad-m/github-push-action@master
+        uses: ad-m/github-push-action@main
```

```suggestion
        uses: ad-m/github-push-action@master
```
The branch reference here shouldn't be updated since it is for another repo.